aries-site/src/tests/accessibility/axe.js

```diff
@@ -24,7 +24,23 @@ test('should check Home page', async t => {
   // Only need to run in one browser
   if (t.browser.name === 'Chrome') {
     const axeContext = {
-      exclude: [],
+      exclude: [
```

Comment:
Are these the label exclusions?

Reply:
yes these are the labels that were in the front tiles that I had to exclude I wasnt sure if this was a good approach for now

Reply:
Great, maybe @halocline can give some advice of how to achieve labels in the DOM but visually hidden? would like to avoid excluding things from the tests wherever possible.

Reply:
yes! Totally forgot to discuss that with @halocline we can talk about that tomorrow

Reply:
It was the adding the `aria-label` instead of excluding these from the tests I just added these labels where they needed to be! @taysea @halocline